components/Activitylist.js

```diff
@@ -40,6 +47,43 @@ export default function Activitylist({
     }
   }
 
+  function handleAddFavoriteButton(id) {
+    const activitySuggestionCard = activityCards.find((card) => card.id === id);
+
+    if (activitySuggestionCard) {
```

Comment:
Just wondering what would happen if you would scrap these two lines? Aren't they redundant? i'm asking 'cause you're looking for a card, even though the button is placed on a card, so maybe a case where a card cannot be found will never happen...🤔